.buildkite/lint.rayci.yml

```diff
@@ -11,3 +11,9 @@ steps:
       - pip install -c python/requirements_compiled.txt -r python/requirements/lint-requirements.txt
       - FORMAT_SH_PRINT_DIFF=1 ./ci/lint/format.sh --all-scripts
     depends_on: forge
+
+  - label: "lint: untested code snippet"
+    commands:
+      - pip install -c python/requirements_compiled.txt semgrep
+      - semgrep ci --config semgrep.yml
+    depends_on: forge
```

Comment:
What does this mean/do?

Reply:
forge is a minimalist docker image in CI-v2. It get builds within 2 minutes instead of 20 minutes like old CI. Consequently, lint in new CI-v2 will finishes in 3 minutes instead of 27 minutes ;).